README.md

```diff
@@ -44,11 +44,24 @@ If you need to add features to the API, the best is to [`npm link`](https://docs
 Openfisca
 ---------
 
-```sh
+Openfisca relies on Python 2. You should [install it in a `virtualenv`](https://doc.openfisca.fr/for_developers.html#create-a-virtualenv):
+
+```bash
 cd mes-aides-ui
-npm run install-openfisca # ou pip install --upgrade -r openfisca/requirements.txt si vous utilisez un environnement virtuel
+virtualenv  --python=python2 .venv # To create your virtualenv in ./.venv (a hidden folder)
```

Comment:
I don't get it. OpenFisca says 2.7. In which cases 2.7 would fail but 2 wouldn't?